### PR TITLE
Pipenv

### PR DESCRIPTION
**Pull request by kayiwa**
after the experience at the lae-hackathon switching to the simpler for our needs pipenv.

### CODE REVIEW

**Comment by kayiwa** [comment]
ping?

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-ansible==2.7.5
+ansible==2.7.6
```
Comment (jrgriffiniii) [inline requirements.txt]: 👍

File: README.md [hunk 2]
```diff
@@ -15,51 +15,24 @@ $ vagrant box add --name princeton_box images/ubuntu-16.04.virtualbox.box
...
+ * `brew cask install vagrant`
+ * `brew install python`
+ * `brew install pipenv`
 
 ## Prerequisites of Ubuntu Bionic
 
  * `sudo add-apt-repository multiverse && sudo apt -y update`
  * `sudo apt -y install virtualbox`
- * `sudo apt-get install vagrant`
- * `git clone https://github.com/pyenv/pyenv.git ~/.pyenv`
```
Comment (jrgriffiniii) [inline README.md]: Not a criticism, but is there a reason why pyenv is deprecated entirely?  From the documentation, it seems as if pipenv and pyenv can coexist in an operating system environment: https://pipenv.readthedocs.io/en/latest/#pipenv-features

Comment (kayiwa) [reply README.md]: simplicity mostly. We aren't building python libraries in this repo. If one isn't a native python dev having less steps to get to writing your tests is a good thing (IMO). pipenv shell will install ansible and molecule in your virtual environment and you can "get on with it".

**Comment by hackmastera** [comment]
"command not found: molecule"

```
10:20:01 princeton_ansible pipenv % pipenv shell
Creating a virtualenv for this project…
Pipfile: /Users/headleya/src/princeton_ansible/Pipfile
Using /usr/local/bin/python3 (3.7.2) to create virtualenv…
⠧ Creating virtual environment...Using base prefix '/usr/local/Cellar/python/3.7.2_1/Frameworks/Python.framework/Versions/3.7'
New python executable in /Users/headleya/.local/share/virtualenvs/princeton_ansible-xwTyWoL_/bin/python3.7
Also creating executable in /Users/headleya/.local/share/virtualenvs/princeton_ansible-xwTyWoL_/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /usr/local/bin/python3

✔ Successfully created virtual environment! 
Virtualenv location: /Users/headleya/.local/share/virtualenvs/princeton_ansible-xwTyWoL_
Launching subshell in virtual environment…
 . /Users/headleya/.local/share/virtualenvs/princeton_ansible-xwTyWoL_/bin/activate
10:20:24 princeton_ansible pipenv %  . /Users/headleya/.local/share/virtualenvs/princeton_ansible-xwTyWoL_/bin/activate
(princeton_ansible) 10:20:24 princeton_ansible pipenv % molecule test
zsh: command not found: molecule
```

**Comment by jrgriffiniii** [comment]
I experienced the exact same error @hackmastera , I assumed that it was a local problem.

**Comment by jrgriffiniii** [comment]
As recommended by @kayiwa, `pipenv sync` (only) followed by `pipenv shell` seems to have worked for me in getting `molecule test` to run

**Comment by hackmastera** [comment]
yep that worked for me too. there's a failing test but I assume that's known?

**Comment by jrgriffiniii** [comment]
@hackmastera I did not find any tests fail locally, and everything seems to have passed on Travis CI: https://travis-ci.org/pulibrary/princeton_ansible/builds/483457183?utm_source=github_status&utm_medium=notification

What did you encounter?

**Comment by hackmastera** [comment]
https://github.com/pulibrary/princeton_ansible/issues/432

**Pull request merged**